ui/app/templates/vault/cluster/init.hbs

```diff
@@ -31,32 +31,37 @@
           </p>
         </div>
         <div
-          class="message is-list is-highlight has-copy-button"
+          class="message is-list is-highlight"
           tabindex="-1"
         >
-          <HoverCopyButton
```

Comment:
👋 byeee HoverCopyButton - wonder if we can get rid of these in the web CLI now too and then just delete the component... something to do for later!